src/content/app/custom-download/components/checkbox-grid/CheckboxGrid.scss

```diff
@@ -29,6 +29,11 @@
   margin-bottom: 5px;
 }
 
+.checkbox {
+  height: 14px;
+  width: 14px;
+}
```

Comment:
So, now that we should be able to customise the size of the checkbox through CSS variables, this class becomes unnecessary. The parent, `checkboxGridContainer`, can define the value of the variable that controls the checkbox size. Only I have a suspicion that we shouldn't customise checkbox size here at all; but simply use the default size of the checkbox. Since it's in the custom download section, and Andrea doesn't care about custom download any longer (she says it will likely morph into some other tool), we can just remove this class from here (and from the tsx that's using it).

Reply:
✅